backend/routes/api/export.js

```diff
@@ -243,7 +244,8 @@ function getFeatureForDateRange(dateRange) {
  * @returns {Object} Object containing gate start time, end time, and same every year values
  */
 function getGateDisplayValues(dateRange, gateDetail, annualData) {
-  const isGateType = dateRange.dateType.name === "Operating";
+  const isGateType =
+    dateRange.dateType.strapiDateTypeId === DATE_TYPE.PARK_GATE_OPEN;
```

Comment:
I really don't like the idea of tying logic to the `strapiDateTypeId` field because it's poorly named, but I create CMS-1341 and CMS-1342 to rename the well known key fields, so this is good for now.